src/parse-stream.js

```diff
@@ -5,6 +5,21 @@ import Stream from '@videojs/vhs-utils/es/stream.js';
 
 const TAB = String.fromCharCode(0x09);
 
+const parseByterange = function(byterangeString) {
```

Comment:
Moved all byterange parsing into here as we do this for `map`, `byterange`, and not `part`